### PR TITLE
Issue 721 update webcam relay

### PR DESCRIPTION
**Pull request by pabbathishyam**
The following updates:
1. Script and Notes on Live Streaming on YouTube.
2. Updated the notes on Relay.
3. Added P6 to Device Lookup Table.
4. Added ctags and cscope.out to .gitignore.

### CODE REVIEW

File: software/controller/integration_tests/relay8/notes.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Here are instructions to build the sainsmartrelay command line utility.
+
+    1. Clone the source code from GitHub - "git clone https://github.com/labbots/SainsmartUsbRelay.git".
+    2. Install any depebencies libraries listed in GitHub page of SainsmartUsbRelay.
```
Comment (asmodai27) [inline software/controller/integration_tests/relay8/notes.txt:4]: Typo on "dependency"

File: software/controller/integration_tests/webcam/notes.txt [hunk 2]
```diff
@@ -31,4 +44,19 @@ ffmpeg \
...
+- Wait 24 hours for "Live streaming" to be enabled on your account
+- Go to YouTube Dashboard: https://www.youtube.com/live_dashboard?nv=1
+- Change category to "Nonprofits & Activism" and privacy to Unlisted
+- Copy "Stream name/key" and paste it in the KEY variable of the stream-to-youtube.sh script
+- Add execute permissions to the script: chmod +x stream-to-youtube.sh
+- Run the script with the KEY passed as the argument: ./stream-to-youtube.sh 8yuc-2eb2-74e9-tmb6-2cm6 (this is an example key)
+- Check that on the YouTube Dashboard that the status has gone from OFFLINE to STARTING to LIVE
+- The URL for the stream is available at the bottom of the page
+
+- There are two webcams are connected to Martin's setup, the server name is bigcheese.
```
Comment (asmodai27) [inline software/controller/integration_tests/webcam/notes.txt:59]: Redundant phrasing: "There are [...] are"

**Review by asmodai27 — COMMENTED** [review]
Will test all of this myself in a couple days (requesting YouTube streaming access). For now, I suggested some minor edits.

File: software/controller/integration_tests/relay8/notes.txt (hunk 1 above)
Comment (pabbathishyam) [reply software/controller/integration_tests/relay8/notes.txt]: Done

File: software/controller/integration_tests/webcam/notes.txt (hunk 2 above)
Comment (pabbathishyam) [reply software/controller/integration_tests/webcam/notes.txt]: Thanks for catching the typo. I corrected.

**Review by asmodai27 — APPROVED** [review]
Webcam works fine for me and everything is straightforward.
The relay is a work in progress but stated as such in the documentation, and this looks good to me.

**Pull request merged**